prow/plugins/label/label_test.go

```diff
@@ -148,6 +148,7 @@ func TestLabel(t *testing.T) {
 			expectedNewLabels:     formatLabels(),
 			expectedRemovedLabels: []string{},
 			commenter:             orgMember,
+			expectedBotComment:    true,
```

Comment:
Nit: Can we have a testcase where we actually see the correct comment being made and not just any comment?

Reply:
Sure thing! I think more tests are always a good thing.

See line 493 in label_test.go.

Thank you!